query.go

```diff
@@ -77,6 +77,7 @@ func NewQuery(host, user, source, catalog, schema, query string) (*Query, error)
 	q := &Query{
 		host:    host,
 		user:    user,
+		source:  source,
```

Comment:
It is pulled in from Line 60 in the func NewQuery already, there is the default value set in line 65 - 67, it just wasn't actually being set in the Query struct previously.

I also added the http header in the request as well, now that it is set in the query struct from the initial NewQuery func initialization.

Cheers :)